README.md

```diff
@@ -55,6 +62,35 @@ $rollout = $this->getServiceLocator()->get('zf2_rollout');
 
 Refer to the documentation of [opensoft/rollout](https://github.com/opensoft/rollout) for more information on how to use the library.
 
+# Description
+
+If you want to display the description of a feature in your views. You need to
+
+##### 1. Add the configuration
+
+```php
+<?php
+return [
+    'rollout' => [
+        
+        //Describes the features with a description - not required -
+        'features' => [
+            'feature_1' => [
+                'description' => '' 
+            ]
+        ]
+    ],
+];
+```
+
+##### 2. Use the view helper
+
+To display the description in a view you have to call the view helper : **rollout_description**. If the description is not found in the configuration, *null* will be returned.
```

Comment:
*null* => an empty string

Reply:
I though to return an empty string too but with a null the code which call the helper can do a business on the null.

Reply:
it's a view helper, so what kind of business you think of doing with a null?

Reply:
if null => N/A

Reply:
And so if someone puts an empty string it will show empty? Seriously? If you really want to show N/A (which honestly would be ugly), then simply add a second parameter where you can provide a default value for the return if no description is available